### PR TITLE
GMPE Atkinson 2010 (Subclass of Boore and Atkinson 2010, modified for Hawaii)

### PR DESCRIPTION
**Pull request by klunk386**
Still missing the tables to implement tests (coming soon....)

NOTE: I accidentally created a branch within another branch still under review (Pankow and Pechmann). Do not consider these other files, which belong to a different pull request...

### CODE REVIEW

**Comment by mmpagani** [comment]
The implementation of the adjusted `get_mean_and_stddevs` seems okay to me. Please remove the unrelated files. I'll complete the review once the verification tables will be also available.

**Comment by daniviga** [comment]
https://github.com/gem/oq-engine/pull/2973 has been merged. Please sync with master and make sure travis is executing hazardlib tests.

File: openquake/hazardlib/gsim/boore_atkinson_2008.py [hunk 1]
```diff
@@ -349,3 +349,78 @@ def _compute_non_linear_term(self, pga4nl, bnl):
...
+    #: see equation 2, pag 106.
+    DEFINED_FOR_STANDARD_DEVIATION_TYPES = set([
+        const.StdDev.TOTAL
+    ])
+
+    # Adding hypocentral depth as required rupture parameter
+    REQUIRES_RUPTURE_PARAMETERS = set(('hypo_depth'))
+
+
+    def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
```
Comment (daniviga) [inline openquake/hazardlib/gsim/boore_atkinson_2008.py]: Too many blank lines on top of a function

File: openquake/hazardlib/gsim/boore_atkinson_2008.py [hunk 2]
```diff
@@ -349,3 +349,78 @@ def _compute_non_linear_term(self, pga4nl, bnl):
...
+        """
+        Using a frequency dependent correction for the mean ground motion.
+        Standard deviation is fixed.
+        """
+
+        base = super(Atkinson2010Hawaii, self)
+        mean, stddevs = base.get_mean_and_stddevs(sites, rup, dists, imt, stddev_types)
```
Comment (daniviga) [inline openquake/hazardlib/gsim/boore_atkinson_2008.py]: Line too long

File: openquake/hazardlib/gsim/boore_atkinson_2008.py [hunk 3]
```diff
@@ -349,3 +349,78 @@ def _compute_non_linear_term(self, pga4nl, bnl):
...
+        # Defining frequency
+        if imt == PGA():
+            freq = 50.0
+        elif imt == PGV():
+            freq = 2.0
+        else:
+            freq = 1./imt.period
+
+        # Equation 3 of Atkinson (2010)
+        x1 = np.min([-0.18+0.17*np.log10(freq),0])
```
Comment (daniviga) [inline openquake/hazardlib/gsim/boore_atkinson_2008.py]: missing whitespace after ','

File: openquake/hazardlib/gsim/boore_atkinson_2008.py [hunk 4]
```diff
@@ -349,3 +349,78 @@ def _compute_non_linear_term(self, pga4nl, bnl):
...
+        """
+        assert all(stddev_type in self.DEFINED_FOR_STANDARD_DEVIATION_TYPES
+                   for stddev_type in stddev_types)
+
+        # Using a frequency independent value of sigma as recommended
+        # in the caption of Table 2 of Atkinson (2010)
+        stddevs = [0.26/np.log10(np.e) + np.zeros(num_sites)]
+
+        return stddevs
+
```
Comment (daniviga) [inline openquake/hazardlib/gsim/boore_atkinson_2008.py]: blank line at end of file

File: openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py [hunk 5]
```diff
@@ -50,4 +50,18 @@ def test_std_inter(self):
                     
     def test_std_total_strike_slip(self):
         self.check('NGA/BA08/BA08_SIGTM_SS.csv',
-                    max_discrep_percentage=0.1)
\ No newline at end of file
+                    max_discrep_percentage=0.1)
+
+
+class Atkinson2010HawaiiTestCase(BaseGSIMTestCase):
+    GSIM_CLASS = Atkinson2010Hawaii
```
Comment (daniviga) [inline openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py]: `Atkinson2010Hawaii` hasn't been imported

File: openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py [hunk 6]
```diff
@@ -50,4 +50,18 @@ def test_std_inter(self):
...
+    
+    # Test data were kindly provided by Gail Atkinson
+
+    def test_mean(self):
+        self.check('NGA/BA08/A10H_MEDIAN.csv',
+                    max_discrep_percentage=0.6)
```
Comment (daniviga) [inline openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py]: over indented by 1 char

File: openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py [hunk 7]
```diff
@@ -50,4 +50,18 @@ def test_std_inter(self):
...
+
+    def test_std_total(self):
+        self.check('NGA/BA08/A10H_STD_TOTAL.csv',
+                    max_discrep_percentage=0.1)
```
Comment (daniviga) [inline openquake/hazardlib/tests/gsim/boore_atkinson_2008_test.py]: over indented by 1 char

**Comment by daniviga** [comment]
https://github.com/gem/oq-engine/pull/2973 has been merged. Please sync with master and make sure travis is executing hazardlib tests.

**Comment by micheles** [comment]
Please fix the PEP 8 issues, for the rest LGTM

**Comment by mmpagani** [comment]
You did not address the suggestion of a tapering provided by Bruce https://github.com/gem/oq-engine/issues/2988

**Comment by mmpagani** [comment]
LGTM

**Pull request merged**